crates/biome_grit_formatter/Cargo.toml

```diff
@@ -13,21 +13,13 @@ version              = "0.0.0"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-biome_diagnostics = { workspace = true }
 biome_formatter   = { workspace = true }
 biome_grit_syntax = { workspace = true }
 biome_rowan       = { workspace = true }
-biome_suppression = { workspace = true }
 
 [dev-dependencies]
-biome_formatter_test = { path = "../biome_formatter_test" }
-biome_grit_parser    = { path = "../biome_grit_parser" }
-biome_parser         = { path = "../biome_parser" }
-biome_service        = { path = "../biome_service" }
-countme              = { workspace = true, features = ["enable"] }
-serde                = { workspace = true, features = ["derive"] }
-serde_json           = { workspace = true }
-tests_macros         = { path = "../tests_macros" }
+serde      = { workspace = true, features = ["derive"] }
+serde_json = { workspace = true }
```

Comment:
I cleared all the unused dependencies (those that weren't imported) except for `serde`. I hope `udeps` does not complain again.